src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -62,8 +104,25 @@ const getNGrams = (words, n) => {
   return nGrams;
 };
 
-const filteredKeywords = computed(() => {
-  return nGrams.value[selectedNGram.value] || [];
+const sortedNGramFilters = computed(() => {
+  return selectedNGramFilters.value.sort((a, b) => {
+    const aNum = parseInt(a.split('-')[0], 10);
+    const bNum = parseInt(b.split('-')[0], 10);
+    return aNum - bNum;
+  });
+});
+const sortedNGrams = computed(() => {
+  // Sort nGrams by key
+  return Object.keys(nGrams.value)
+    .sort((a, b) => {
+      const aNum = parseInt(a.split('-')[0], 10);
+      const bNum = parseInt(b.split('-')[0], 10);
+      return aNum - bNum;
+    })
```

Comment:
The `sort` part can be moved to a separate function. Both `sortedNGramFilters` and `sortedNGrams` have the same sort implementation.